charts/logging-pipeline-plumber/values.yaml

```diff
@@ -1,11 +1,13 @@
-# Default values for rancher-logging-explorer.
+# Default values for logging-pipeline-plumber.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
 
+namespace: default
```

Comment:
do not specify a default namespace, users can define it with the -n flag

Reply:
https://github.com/MrSupiri/rancher-logging-explorer/pull/31#discussion_r677699415
same issue